components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/sequence/impl/DefaultSequenceHandlerUtils.java

```diff
@@ -335,4 +335,32 @@ public static String getSpRoleClaimUri(ApplicationConfig appConfig) {
 
         return spRoleClaimUri;
     }
+
+    /**
+     * Get standard role claim uri for the given service provider role uri.
+     *
+     * @param context        AuthenticationContext.
+     * @param spRoleUri      Service Provider role claim URI.
+     * @param sequenceConfig SequenceConfig.
+     * @return Standard role claim uri for the given spRoleUri.
+     * @throws FrameworkException If an error occurred while reading standard role uri.
+     */
+    public static String getStandardRoleClaimUri(AuthenticationContext context, String spRoleUri,
+                                                 SequenceConfig sequenceConfig)
+            throws FrameworkException {
+
+        String spStandardDialect = DefaultSequenceHandlerUtils.getSPStandardDialect(context);
+        if (spStandardDialect != null && DefaultSequenceHandlerUtils.isLocalClaimDialect(context)) {
+            spRoleUri = DefaultSequenceHandlerUtils.getStandardRoleClaimURI(spStandardDialect,
+                    context.getTenantDomain());
+
```

Comment:
Lets not add extra white spaces

Reply:
fixed https://github.com/wso2/carbon-identity-framework/pull/3747